undelete

```diff
@@ -156,6 +139,7 @@ def main():
             rv = 1
     return rv
 
+
```

Comment:
Why so many extra blank lines?
(Sorry for the super sparse review; I'm on vacation this week and just skimming.)

Reply:
Flake8 compliance.